### DISCUSSION TITLE
IterationAdaptiveDT unused parameters

### DISCUSSION THREAD

**Discussion by fervaldies** [idaholab/moose#23395] [Q&A General] [answered]
Hello, I am trying to run my simulation with the IterationAdaptiveDT (look at the part of the Executioner pasted below) and I receive and error saying that linear_iteraction_ratio, optimal_iteractions, and iteraction_window are "unused parameters". Is this because I need to add anything else in the code apart from this part in the Executioner named "TimeStepper"? Thank you in advance.
-----------------------------------------------------------------
-----------------------------------------------------------------
-----------------------------------------------------------------

[Executioner]
  # Transient (time-dependent) problem
  type = Transient

  # Solver
  solve_type = NEWTON

  #Options for PETSc (how to solve linear equations)
  automatic_scaling = false
  petsc_options = '-snes_converged_reason -ksp_converged_reason'
  petsc_options_iname = '-pc_type -pc_hypre_type -ksp_gmres_restart -pc_hypre_boomeramg_strong_threshold -pc_hypre_boomeramg_agg_nl -pc_hypre_boomeramg_agg_num_paths -pc_hypre_boomeramg_max_levels -pc_hypre_boomeramg_coarsen_type -pc_hypre_boomeramg_interp_type -pc_hypre_boomeramg_P_max -pc_hypre_boomeramg_truncfactor -pc_hypre_boomeramg_print_statistics'
  petsc_options_value = 'hypre boomeramg 51 0.7 4 5 25 PMIS ext+i 2 0.3 0'

  # Solver tolerances
  l_max_its = 50
  #l_tot = 1e-4 #1e-6
  nl_max_its = 50
  nl_rel_tol = 1e-6
  nl_abs_tol = 1e-6
  nl_forced_its = 1
  line_search = 'bt' # 'none'/'bt'

  # Time variables
  start_time = 0.0
  end_time = 15000.0
  dtmin = 1e-6
  dtmax = 0.1

  [./TimeStepper]
    type = IterationAdaptiveDT
    growth_factor = 2
    cutback_factor = 0.5
    linear_iteraction_ratio = 1000
    optimal_iteractions = 8 #12
    iteraction_window = 3
    dt = 0.1
  [../]
[]

**Comment by GiudGiud** [comment, marked as answer]
Hello

nope it s just spelled iteration not iteraction 
Once that’s fixed it should stop complaining about unused parameters. 

Guillaume

**Reply by fervaldies** [reply]
Oh, thank you very much! haha my bad, I was confused with interaction. Now it is working
